website/versioned_docs/version-5.x/module-types.md

```diff
@@ -0,0 +1,51 @@
+---
+id: module-types
+title: single-spa microfrontend types
+sidebar_label: Conceptual Module Types
+---
+
+# Concept: single-spa microfrontend types
+
+Single-spa has [different categories](/docs/microfrontends-concept#types-of-microfrontends) of microfrontends. It is up to you where and how you use each of them. However, the single-spa core team has [recommendations](/docs/recommended-setup/#applications-versus-parcels-versus-utility-modules).
+
+Here is how each single-spa microfrontend works conceptually. This information should help you understand our [recommendations](/docs/recommended-setup/#applications-versus-parcels-versus-utility-modules).
+
+| application                       | parcel                               | utility                              |
```

Comment:
I think this table would make more sense if there were a column that explained what each row was. For example, the first row could be called "routing".

```suggestion
| Topic                | application                       | parcel                               | utility                              |
```